tests/integrationv2/common.py

```diff
@@ -38,39 +39,28 @@ def pq_enabled():
 
 class AvailablePorts(object):
     """
-    This iterator will atomically return the next number.
+    This iterator will atomically return the next available port.
     This is useful when running multiple tests in parallel
     that all need unique port numbers.
     """
 
-    def __init__(self, low=8000, high=30000):
```

Comment:
nit: It looks like some of the imports are no longer needed after removing this.